packages/frontend/src/styles/blueprint-core.css

```diff
@@ -8986,12 +8986,7 @@ label.bp4-label:not(.bp4-inline) .bp4-popover-target {
     right: 12px;
     top: 12px;
 }
-.bp4-html-select.bp4-fill,
-.bp4-html-select.bp4-fill select,
-.bp4-select.bp4-fill,
-.bp4-select.bp4-fill select {
-    width: 100%;
-}
```

Comment:
could we revert this change please? I'm unsure if we're using Blueprint `select`s in other places, so it's less risky to keep until we fully migrate 👍